### ISSUE TITLE
Delay before item can be dropped

### ISSUE THREAD

**Issue by binary-koan** [isaacHagoel/svelte-dnd-action#212]
I tried out [the example in the REPL](https://svelte.dev/repl/e2ef044af75c4b16b424b8219fb31fd9?version=3.22.2) and was surprised to find that if I drag and drop an item quickly before the other objects have been rearranged, the item stays in the same place instead of being dropped into the new container. In order to move something you need to drag -> wait a second for the other objects to move -> drop.

This is especially noticeable in the "board" demo; I can't quickly flick an item into "doing". I assumed it was because of the default 300ms animation time, but even after setting that to zero I can't drop immediately into the new container.

I would expect the drag and drop widget to compute the final position and reorganize the items when I release the mouse - most other times I've used drag and drop I've been able to quickly drag and release without waiting for the list to catch up.

Apologies if this is intentional or if there's already a way to turn it off!

Tried in Firefox and Edge on Windows 10

**Comment by yjp20** [comment]
I'm not the author, but I'm pretty sure this is because of the flipDuration. Because flip is handled by svelte internally, if you change the state during the flip animation the animation becomes unsmooth. So, the library refreshes in intervals longer than the flipduration to work around the problem.

Currently, you can either reduce the flipDuration or turn it off entirely.

**Comment by yjp20** [comment]
You have to set the flip duration to zero on all dropzones

**Comment by binary-koan** [comment]
I tried that but got the same result - [see repl](https://svelte.dev/repl/7ce4bd861e1446adb03f98bd4438cc75?version=3.31.2) (top one is the demo code, second one has 0ms duration, third one has no flip animation at all - not sure if it makes a difference)

I can still reproduce the behaviour on all three of those examples. Finally got a gif of it -

Releasing too fast (item does not move):

![Jan-08-2021 11-26-58 PM](https://user-images.githubusercontent.com/1077405/104026536-69e91b80-5209-11eb-93dc-515884276849.gif)

Releasing slower (item does move):

![Jan-08-2021 11-25-17 PM](https://user-images.githubusercontent.com/1077405/104026555-740b1a00-5209-11eb-96f0-a2ae2b0a3d29.gif)

**Comment by isaacHagoel** [comment]
The library samples the status of the dragged element in fixed intervals even if flip duration is zero.
In this specific case, I also think it is a feature and not a bug 😄 
User interfaces are all about communicating back and forth with the user, and in this case if the user let go of the item super quickly, cancelling the operation makes a lot of sense to me.
Open for being convinced otherwise.

**Comment by binary-koan** [comment]
I figured it out! For reference: In the above repl I'm not specifying the `type` on the drop zones, so the items are actually draggable between all three boards. And the observe interval (the delay I was seeing) depends on the *highest animation duration* out of all drop zones, i.e. when I removed the 300ms board from the page it no longer delayed dragging and dropping on the other two boards.

It feels nice and snappy after changing `flipDurationMs` to around 100ms on all the drop zones. Thanks for the pointers, I should've read the docs more carefully 🤦‍♂️
